nixos/modules/virtualisation/kubevirt.nix

```diff
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
not sure if thats still needed. from a glance I cant see anything which would require lib